### PR TITLE
Fully stateless API

### PR DESCRIPTION
**Pull request by nicklawls**

### CODE REVIEW

**Comment by ashishsc** [comment]
you totally didn't use ```elm-format@exp```

File: src/Main.elm [hunk 1]
```diff
@@ -70,11 +76,38 @@ update msg model =
         FeedbackCancelled ->
             ( { model | timesCancelled = model.timesCancelled + 1, feedback = Feedback.init }, Cmd.none )
 
+        ChooseReason r ->
```
Comment (ashishsc) [inline src/Main.elm]: Seems like you would want functions for dealing with this in case you *don't* want to handle the logic yourself

**Comment by nicklawls** [comment]
Got `elm-format@exp` back in, tried the two-module design, and cleaned up a few things. lmk what you think

**Comment by ashishsc** [comment]
Nice looks good to me!

**Pull request merged**